api/tests/unit/application/certification-report/index_test.js

```diff
@@ -17,4 +18,35 @@ describe('Unit | Application | Certifications Report | Route', function() {
     // then
     expect(response.statusCode).to.equal(200);
   });
+
+  describe('POST /api/certification-reports/{id}/abort', function() {
+
+    it('allow access if the session of the logged used is authorized', async function() {
```

Comment:
**Suggestion**: "Returns HTTP 200 if the logged user can access the session". Ou un truc du genre car ce que l'on vérifie ici (dans le expect) c'est le code http de la réponse